src/Mod/Surface/Gui/Workbench.cpp

```diff
@@ -74,8 +76,10 @@ Gui::ToolBarItem* Workbench::setupToolBars() const
              << "Surface_GeomFillSurface"
              << "Surface_Sections"
              << "Surface_ExtendFace"
-             << "Surface_CurveOnMesh";
-/*
+             << "Surface_CurveOnMesh"
+             << "BlendCurve";
+

```

Comment:
Please remove this empty line

Reply:
Thanks. Do you mind also removing any trailing whitespace that this commit introduces? TIA